### ISSUE TITLE
Is there a Bug in Datafeeder Module of GeoOrchestra?

### ISSUE THREAD

**Issue by haytastan** [georchestra/docker#282]
While trying to import (feed) data using datafeeder GUI of geOrhestra (i.e at the secont step: 
https://mydomain/import/f8d06d81-2087-4377-9e68-e7f53ebdfaf6/step/2) I am supposed to "select one or more tags that fit my dataset", but nothing is listed in the textbox as shown below.

 
![bug_of_datafeeder](https://github.com/georchestra/docker/assets/7123870/ad4b9318-1bad-4494-94e1-6869130e9d60)

**Comment by jeanmi151** [comment]
Hello,
If you are using the 23.0 branch , you probably didn't change the thesaurus url in the datafeeder configuration, this file : https://github.com/georchestra/datadir/blob/c939dab1816fe53fb91adaed584512c4fa257e36/datafeeder/frontend-config.json#L26

**Comment by haytastan** [comment]
Thanks for your reply. 

I changed the thesaurus url in the datafeeder configuration file (frontend-config.json file) which is in the folder (i.e. docker/config/datafeeder) by replacing the **georchestra-127-0-1-1.traefik.me**  with **mydomain**.

I stopped, pruned (system&volume) and started docker using the following ubuntu commands:

- **sudo docker compose -f docker-compose.yml -f docker-compose.override.yml -f docker-compose.gwc.yml -f docker-compose.atlas.yml down**
- **sudo docker system prune -a**
- **sudo docker volume prune -a** 
- **sudo docker compose -f docker-compose.yml -f docker-compose.override.yml -f docker-compose.gwc.yml -f docker-compose.atlas.yml up -d**

As a result **the problem is not solved**. 
Am I supposed to define the tags (to be selected ) anywhere in GeOrchestra, or?
Regards,
